api/src/main/scala/com/rasterfoundry/granary/options/AuthOptions.scala

```diff
@@ -0,0 +1,40 @@
+package com.rasterfoundry.granary.api.options
+
+import com.rasterfoundry.granary.api.AuthConfig
+
+import cats.implicits._
+import com.monovore.decline._
+
+import java.util.UUID
+
+trait AuthOptions {
+
+  private val authEnabledHelp = "Whether to enable support for user authentication"
+  private val authEnabled     = Opts.flag("with-auth", help = authEnabledHelp).orFalse
+
+  private val anonymousUserIdHelp =
+    "UUID corresponding to the 'anonymous' user when auth is not enabled"
+
+  private val anonymousUserId: Opts[UUID] =
+    Opts.option[UUID]("anonymous-user-id", help = anonymousUserIdHelp) orElse Opts.env[UUID](
```

Comment:
I like the way these environment variable settings worked out

Reply:
I was also really pleased about it!